apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.Snippets/DatastoreClientSnippets.cs

```diff
@@ -39,9 +39,10 @@ public void Lookup()
         {
             string projectId = _fixture.ProjectId;
             string namespaceId = _fixture.NamespaceId;
+            string databaseId = _fixture.DatabaseId;
 
             // Snippet: Lookup(*,*,*,*)
-            KeyFactory keyFactory = new KeyFactory(projectId, namespaceId, "book");
+            KeyFactory keyFactory = new KeyFactory(projectId, namespaceId, databaseId, "book");
```

Comment:
Do we expect that using a database ID will be common practice now? It's not clear to me whether we should really make all our snippets show this. (We don't in Firestore for example.)